cmd/hactar/commands/commands.go

```diff
@@ -78,20 +78,30 @@ var StartCommand = &cli.Command{
 		// display token and URL
 		token.DisplayTokens()
 		url.DisplayUrl()
-		// save node to backend
-		node, resp, err := hactarClient.Nodes.Add(hactar.Node{
-			Token: token.ReadNodeTokenFromFile(),
-			Node: hactar.NodeInfo{
-				Address: actorAddress,
-				Url:     url.GetUrl(),
-			},
-		})
-		if err != nil {
-			log.Error("Adding new node failed.", err)
-			return nil
-		} else if resp != nil && resp.StatusCode == http.StatusOK {
-			log.Info(fmt.Sprintf("New node added, url: %s address: %s", node.Node.Url, node.Node.Address))
+		// this check for existing nodes is just placeholder
+		nodes, _, err := hactarClient.Nodes.GetAllNodes()
+		if err == nil {
+			// TMP condition
```

Comment:
Elaborate please :) Backend should check if it already exists or not, so we should be good here

Reply:
Fixed now, you were right. I had everything I need there :+1: